src/components/buttons/rainbow-button/RainbowButton.js

```diff
@@ -1,8 +1,8 @@
 import MaskedView from '@react-native-community/masked-view';
 import React from 'react';
-import FastImage from 'react-native-fast-image';
 import styled from 'styled-components/primitives';
 import AddCashIconSource from '../../../assets/addCashIcon.png';
+import ImgixImage from '../../ImgixImage';
```

Comment:
Can you use some alias here like `@images/ImgixImage`

Reply:
Thanks for checking it out 🙏 That sounds great!

Do you think it would make sense to have a dedicated directory for image-style components like:

```diff
"@rainbow-me/apollo": ["./src/apollo"],
"@rainbow-me/assets": ["./src/assets"],
"@rainbow-me/config": ["./src/config"],
+ "@rainbow-me/images": ["./src/components/images"],
```

This way we could probably pull over `ImgixImage`, `ImageWithCachedMetadata` `Avatar` and `ImageAvatar`, as these seem the most used by the wider project. Then we could use:

```typescript
import { ImgixImage, ImageAvatar, ImageWithCachedMetadata, Avatar } from "@rainbow-me/images";
```

Or would it be preferable to use a direct path?

Reply:
Sounds good to me